### PR TITLE
[#241] Add 'Project Details' to top  menu entry

### PR DESCRIPTION
**Pull request by tonythomas01**
Added in the top menu as requested in the bug, and now takes the user to a new page with entire project details!

### CODE REVIEW

**Comment by jaragunde** [comment]
The patch is looking good! Just some comments:
* When there is a lot of projects in the application, a client combo box becomes a must. Sorry for not realizing earlier.
* Please move the JavaScript code in projectDetails.php to a separate .js file, like in tasks.php and js/tasks.js. Many pages still mix php and js code but we want to get rid of that.
* I'm not sure if the details report should open in a new page, but let's keep it like that and let users decide.

**Comment by jaragunde** [comment]
Ah, just another silly comment. Could you place the new menu entry above the "project evaluation" entry, and use lower case for the "details" word, like the other menu entries? :-)

File: web/projectDetails.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+include_once("include/header.php");
+include_once("include/sidebar.php");
+include_once(PHPREPORT_ROOT . '/web/services/WebServicesFunctions.php');
+$user = $_SESSION['user'];
+?>
+
+<script type="text/javascript" src="js/include/ExportableGridPanel.js"></script>
+<script type="text/javascript">
+    var user = "<?php echo $user->getLogin(); ?>";
+</script>
```
Comment (jaragunde) [inline web/projectDetails.php]: I think this parameter doesn't make sense. We want to get the lists of customers and projects regardless of the user. Could you try to remove it?

**Comment by tonythomas01** [comment]
I've rebased it over the other branch. Looks good ?

**Comment by jaragunde** [comment]
Looks great now, thanks!

**Pull request merged**